### PR TITLE
Template consolidation and coding standards

### PR DESCRIPTION
**Pull request by joewhitsitt**
- Coding standard fixes
- Created generic unordered list field template to consolidate.
- Deleted person_education bulleted list template because it is not being used for person_card (full node) styles. Not aware of a grad or admissions content type or person variant using it.

### CODE REVIEW

File: docroot/modules/custom/sitenow_seo/sitenow_seo.module [hunk 1]
```diff
@@ -22,7 +22,7 @@ function sitenow_seo_form_google_analytics_admin_settings_alter(&$form, FormStat
 /**
  * Implements hook_metatags_alter().
  */
-function sitenow_metatags_alter(array &$metatags, array &$context) {
+function sitenow_seo_metatags_alter(array &$metatags, array &$context) {
```
Comment (richardbporter) [inline docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: Wow were these hooks not working then?

Comment (joewhitsitt) [reply docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: i couldn't figure out a way to test this one.

Comment (joewhitsitt) [reply docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: the other was just a couple days live. i moved it from the profile and merged it but copied the function name from the one above.

Comment (richardbporter) [reply docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: I must've goofed this up a long time ago. I think I can test it.

**Comment by joewhitsitt** [comment]
Hoping to get @pyrello to spot check this:
>Deleted person_education bulleted list template because it is not being used for person_card (full node) styles. Not aware of a grad or admissions content type or person variant using it.

File: docroot/modules/custom/sitenow_seo/sitenow_seo.module (hunk 1 above)
Comment (richardbporter) [reply docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: Where are we using a non-AJAX pager anymore?

**Comment by pyrello** [comment]
@joewhitsitt If we are doing something different with this field, it would be in a different content type than person and therefore use different templates. Otherwise, we want to stay in line with how this field is used, so this change should be good for Grad. Not sure about Admissions.

File: docroot/modules/custom/sitenow_seo/sitenow_seo.module (hunk 1 above)
Comment (richardbporter) [reply docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: Original issue https://github.com/uiowa/uiowa/issues/115

Comment (richardbporter) [reply docroot/modules/custom/sitenow_seo/sitenow_seo.module:25]: Oh well. I'm sure its fine. Only needed for non-AJAX paged views but I can't find any.

**Review by richardbporter — APPROVED** [review]

**Pull request merged**